src/Core/AdminConsole/OrganizationFeatures/OrganizationUsers/RemoveOrganizationUserCommand.cs

```diff
@@ -106,21 +114,33 @@ public async Task<List<Tuple<OrganizationUser, string>>> RemoveUsersAsync(Guid o
                     throw new BadRequestException("Only owners can delete other owners.");
                 }
 
+                if (managementStatus.TryGetValue(orgUser.Id, out var isManaged) && isManaged)
+                {
+                    throw new BadRequestException("Managed members cannot be simply removed, their entire individual account must be deleted.");
```

Comment:
Please check with Product/Design about this wording if you haven't already.

Reply:
Thanks for reminding me! I did check and changed the copy.